store/cachemultistore.go

```diff
@@ -10,14 +10,18 @@ import (
 // cacheMultiStore holds many cache-wrapped stores.
 // Implements MultiStore.
 type cacheMultiStore struct {
-	db     CacheKVStore
-	stores map[StoreKey]CacheWrap
+	db         CacheKVStore
+	stores     map[StoreKey]CacheWrap
+	keysByName map[string]StoreKey
 }
 
+var _ CacheMultiStore = cacheMultiStore{}
```

Comment:
Could we stick to one way of doing this?

I think the default is becoming `var _ Queryable = (*rootMultiStore)(nil)`.